### ISSUE TITLE
Single Product Button Settings does not have `Hide Funding Method(s)` field.

### ISSUE THREAD

**Issue by htdat** [woocommerce/woocommerce-gateway-paypal-express-checkout#767]
### Describe the bug

Customers want to hide `Debit or Credit Card` black button in single product pages but there is no option to choose. 

### To reproduce

1. Go to `PayPal Checkout` settings page
1. Select `Checkout on Single Product` option
1. Field `Hide Funding Method(s)` does not display. 
1. `Debit or Credit Card` buttons are still there in single product pages. 

**Screenshots**

![Screen Shot on 2020-06-11 at 13:06:54](https://user-images.githubusercontent.com/10045087/84350914-79a57080-abe4-11ea-8562-e40f4e665e99.png)

### Expected behavior

- Having field `Hide Funding Method(s)` to disable `Debit or Credit Card` buttons in single product pages

**Environment (please complete the following information):**
 - WordPress Version: 5.4.2
 - WooCommerce Version: 4.2.0
 - PayPal (Express) Checkout Plugin Version: 2.0.2 

### Additional details

3024064-zen
3015417-zen 
<details><summary>System status</summary>
  
  ```
  <!--If applicable, paste the system status here. Please ensure you redact or remove any identifying information. -->
  ```
</details>

**Comment by Areahints** [comment]
I'm having similar problems, see issue #527

it's happening on a production site so I've had to use another plugin.

**Comment by jorgeatorres** [comment]
Hi @htdat!

Thanks for the report. "Hide Funding Method(s)" is only shown when "Button Layout" is set to "Vertical". That is, the horizontal layout doesn't support choosing which payment options to show/hide. Is that not working correctly here?

**Comment by mattallan** [comment]
We have an open PR to address this, see https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/pull/758

tl;dr with the new PayPal SDK, the Debit card options are now being displayed on Horizontal layout and we have no option/setting for store managers to hide it (because we only show the Hide Funding Methods field on Vertical layout)

The PR I added makes sure we manually hide the Cards button on Horizontal layout

**Comment by jorgeatorres** [comment]
Now that #774 has been merged, the Cards button should be hidden on the "Horizontal" layout (pending release).
As for "Hide Funding Method(s)" not being present under "Single Product Button Settings", [as explained](https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/767#issuecomment-642902370) it's due to the layout not being set to "Vertical", which is the one layout allowing methods to be hidden.
